assets/src/components/theme-card/index.js

```diff
@@ -72,9 +72,9 @@ export function ThemeCard( { description, ElementName = 'li', homepage, screensh
 							updateOptions( { reader_theme: slug } );
 						} }
 					/>
-					<h3>
+					<h4 className="h3">
```

Comment:
Can we avoid confusing class names like this? If we need to style a `h4` as a `.h3`, then the style is obviously not `h3`-specific and should be renamed.

Reply:
Turns out this was the first use of one of these heading classes, so I removed them and replaced this with a BEM-style class for these headings in c83d149576b05622beed975e78acc6ee744fcd6f